Sources/Extensions/ColletionExtension.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ColletionExtension.swift
+//  HandySwift iOS
+//
+//  Created by Stepanov Pavel on 08/07/2018.
+//  Copyright © 2018 Flinesoft. All rights reserved.
+//
+
+import Foundation
+
+extension Collection {
+    
```

Comment:
This empty line should actually have shown up as a warning if you have SwiftLint installed on your Mac. Please remove it.